lxd/instance/drivers/driver_qemu.go

```diff
@@ -4113,7 +4113,7 @@ func (d *qemu) addDriveConfig(qemuDev map[string]string, bootIndexes map[string]
 		revert := revert.New()
 		defer revert.Fail()
 
-		nodeName := fmt.Sprintf("%s%s", qemuDeviceNamePrefix, escapedDeviceName)
+		nodeName := qemuDeviceNameOrID(qemuDeviceNamePrefix, escapedDeviceName, "", qemuDeviceNameMaxLength)
```

Comment:
Do you think we could just use the value from `blockDev["node-name"]`, defined [here](https://github.com/canonical/lxd/pull/14461/commits/668d666c5777aae921912dd5dc88bee6895f30e9#diff-87040b4a9b898c058e0f889f869c7a0448135f0a7f02db851824f48290c3a2f5R3954), instead of creating a new variable?

Reply:
Are they conceptually the same, because in the disk removal you linked above its called blockDevName rather than nodeName, making me think that the node name and the FD set name can be different conceptually.

But apart from that I can see the benefit of referencing the node name directly as you suggest.

Reply:
Indeed they are conceptually different, the FD name only has to be unique per device and predictable so we can derive it on device removal. And using the node name accomplishes this, as this PR already does but we are computing it again effectively.

Reply:
In this case, I think it makes sense to add `nodeName` for conceptual clarity and readability.